signer/src/bitcoin/rpc.rs

```diff
@@ -275,3 +318,40 @@ impl BitcoinCoreClient {
         Ok(FeeEstimate { sats_per_vbyte })
     }
 }
+
+impl BitcoinInteract for BitcoinCoreClient {
```

Comment:
Any particular reason to re-implement this here when it's on `ApiFallbackClient<BitcoinCoreClient>` in `client.rs`? Just asking, I had removed it as I figured that the only real usage will be via the fallback client and so left only the "internal" implementations here. Of course moving the "actual" `BitcoinInteract` implementations here and calling into them via the fallback-client's implementation is another option -- just more code.

Reply:
Oh I believe I added it because it was necessary for some type bounds. But I might have subsequently changed something that makes it unnecessary now. I'll need to check.

Reply:
Yeah it was necessary in order to use the `SignerContext` with the `BitcoinCoreClient` in the tests.

Reply:
Yeah, got it :+1: